postgresql/db-common-bgwriter-status-9-2.yml

```diff
@@ -0,0 +1,34 @@
+type: db
+data:
+  query: SELECT * FROM pg_stat_bgwriter;
+  dbUrl: jdbc:postgresql://${POSTGRESQL_DB_HOST_PORT}
+  dbDriverClass: org.postgresql.Driver
+  dbUser: ${POSTGRESQL_DB_USER}
+  dbPassword: ${POSTGRESQL_DB_PASSWORD}
+  dbAdditionalConnectionParams: ${POSTGRESQL_DB_ADDITIONAL_PARAMS}
+  #  dbVerticalModel="true" - because output of SHOW STATUS command gives each attribute in its own row
+  # (1st column being attribute name, 2nd column being attribute value)
+  dbVerticalModel: true
+
+require:
+  - className: com.sematext.spm.client.postgresql.PostgreSQLVersionCheck
```

Comment:
Are these 2 attributes applicable only for 9.2 or all version greater than 9.2?

Reply:
Thanks for good question, i check and correct all version range check. This is for all version greater or equals to 9.2